SayTheirNames/Source/View/Hashtag/HashtagView.swift

```diff
@@ -39,10 +39,13 @@ class HashtagView: UIView {
         return label
     }()
 
+    private var originalText: String!
```

Comment:
This is only used in `copyHashtagToClipboard`. Can we move the declaration into that function?

Reply:
From what I understand, pressing down on the button and letting go are two separate calls from the gesture recognizer. So if it were only declared inside the function, when the function got called with .state == .ended, it will have been redeclared and won't have anything to put in the label

Reply:
Can we please get rid of this force unwrap?